src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -223,6 +227,11 @@ Friend Module CompilationUtils
             End If
         End If
 
+#If TEST_OPERATION Then
+        Dim compilationForOperationWalking = VisualBasicCompilation.Create(If(assemblyName, GetUniqueName()), sourceTrees, references, options)
```

Comment:
I was thinking about having each compilation has a different name, but it probably doesn't matter.